src/Model/Consignment/AbstractConsignment.php

```diff
@@ -800,23 +766,23 @@ public function getFullStreet(bool $useStreetAdditionalInfo = false): string
         $fullStreet = $this->getStreet($useStreetAdditionalInfo);
 
         if ($this->getNumber()) {
-            $fullStreet .= ' ' . $this->getNumber();
+            $fullStreet .= ' '.$this->getNumber();
```

Comment:
In dit bestand is string concatenation allemaal zonder de spaties rond de punt...

Reply:
Heb het hele bestand voorzichtig maar even geformatteerd (maar dan zonder mijn alfabetische volgorde dingetje...)